clients/src/main/java/org/apache/kafka/common/network/Selector.java

```diff
@@ -1281,14 +1281,14 @@ private Meter createMeter(Metrics metrics, String groupName,  Map<String, String
 
         /**
          * This method generates `time-total` metrics but has a couple of deficiencies: no `-ns` suffix and no dash between basename
-         * and `time-toal` suffix.
+         * and `time-total` suffix.
          * @deprecated use {{@link #createIOThreadRatioMeter(Metrics, String, Map, String, String)}} for new metrics instead
          */
         @Deprecated
         private Meter createIOThreadRatioMeterLegacy(Metrics metrics, String groupName,  Map<String, String> metricTags,
                 String baseName, String action) {
             MetricName rateMetricName = metrics.metricName(baseName + "-ratio", groupName,
-                    String.format("*Deprecated* The fraction of time the I/O thread spent %s", action), metricTags);
+                    String.format("The fraction of time the I/O thread spent %s", action), metricTags);
```

Comment:
nice finding! Could you add comments to explain why we remove the `deprecated`?

Reply:
Hi, please find it in PR description

Reply:
Added in method javadoc

Reply:
Can we move the comments to here instead of modifying the method docs?

Reply:
Ok, moved to inline comments